src/RxObservableQuery.ts

```diff
@@ -19,7 +19,7 @@ export class RxObservableQuery<T> extends Observable<T> {
     }
   }
 
-  public lift<T, R>(operator: Operator<T, R>): Observable<R> {
+  public lift<R>(operator: Operator<ApolloQueryResult<T>, ApolloQueryResult<R>>): Observable<ApolloQueryResult<R>> {
```

Comment:
I think this is wrong - has broken stuff for me.

Should be something like... 
```ts
public lift<R>(operator: Operator<ApolloQueryResult<T>, R>): Observable<R> {
```

Otherwise `map` etc doesn't work.

I'm currently getting typescript compile errors because of the type mismatch when i use `.map`

```
rxObs.map(result => result.data)  // invalid type, as T is not compatible with ApolloQueryResult<T>
```

See also: 

- https://github.com/ReactiveX/rxjs/blob/master/src/Observable.ts#L67
- https://github.com/ReactiveX/rxjs/blob/master/src/operator/map.ts